modules/mesh_com/package.xml

```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>mesh_com</name>
-  <version>0.2.0</version>
+  <version>0.3.0</version>
   <description>ROS Mesh Node</description>
   <maintainer email="mika.joenpera@unikie.com">joenpera</maintainer>
   <license>TODO: License declaration</license>
```

Comment:
We should probably have a discussion about this (what license) on Thursday? BSD?